internal/runners/activate/activate.go

```diff
@@ -121,6 +122,8 @@ func (r *Activate) run(params *ActivateParams) error {
 				return nil
 			}
 
+			logging.Debug("Currently activated project: %s", activated)
+			logging.Debug("Activated environment: %s", strings.Join(os.Environ(), "\n"))
```

Comment:
This is going to blow up our log file, plus might leak sensitive info for end-users.